x-pack/docs/en/watcher/trigger.asciidoc

```diff
@@ -1,6 +1,9 @@
 [role="xpack"]
-[[trigger]]
-== Triggers
+[[triggers]]
```

Comment:
Not sure if this anchor change was intentional. If so, we should add a redirect for it.

Reply:
I'll revert that one, too.